src/text.rs

```diff
@@ -179,6 +179,43 @@ impl<'a> Span<'a> {
             })
             .filter(|s| s.symbol != "\n")
     }
+
+    /// Patches the style an existing Span, adding modifiers from the given style.
```

Comment:
The patch style changes to patch the style seem like they're unrelated to this change. Is this just based on a previous change that's not yet merged, or included by mistake?

Reply:
Well, it looks like I accidentally pushed stuff from a PR that has already been merged. Is there any way for me to get rid of these commits? I'm not super familiar with using forked repos.

Reply:
in your local, assuming that you have your repo setup as origin and this one as upstream, you can do something like:
```text
git switch main
git pull upstream main
git switch text-style
git rebase main
git push origin text-style --force
```
but handling the rebase with 13 commits is probably going to be more painful than doing it with just 1 or 2.
I'd suggest squashing the commits before doing all of that
```
git rebase --interactive main
```
(choose squash for everything except your first commit)

You may end up having to either skip commits that are already merged (if you don't have the exact same version of that commit) `git rebase --skip` or update your commit to address any incompatible changes. Depending on the number of conflicts, it could be easier to re-do the renames, so it's worth splitting the changes that add new patch style into a separate commit to make that easy.

Reply:
Would you be fine with just migrating your comments to a different PR and closing this one?

Reply:
Actually I'll just try what you said first, and see if it works.